src/commands/edge/channels/drivers.ts

```diff
@@ -26,12 +27,12 @@ export default class ChannelsDriversCommand extends EdgeCommand {
 		const config = {
 			primaryKeyName: 'channelId',
 			sortKeyName: 'version',
-			listTableFieldDefinitions: ['channelId', 'driverId', 'version', 'createdDate', 'lastModifiedDate'],
+			listTableFieldDefinitions: ['name', 'channelId', 'driverId', 'version', 'createdDate', 'lastModifiedDate'],
```

Comment:
It wasn't added in this change, but I am wondering why we're including `channelId` in the output table when it is a required input argument. It's always going to be the same for each row, and this table is already pretty wide.